openapi/openapi.yaml

```diff
@@ -160,6 +160,10 @@ paths:
       parameters:
         - $ref: '#/components/parameters/PageOffset'
         - $ref: '#/components/parameters/PageLimit'
+        - $ref: '#/components/parameters/SearchKeyword'
```

Comment:
We don't offer full text search in public API, only filters by namespace, name and maybe tags.